robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -98,6 +108,50 @@ public static final class ELEVATOR {
     public static final Distance[] ELEVATOR_HEIGHT_SETPOINTS = {};
   }
 
+  public static final class LATERATOR {
+
+    public static final SparkBaseConfig MOTOR_CONFIG_LEFT = new SparkMaxConfig()
+      .idleMode(IdleMode.kBrake)
+      .inverted(false);
+
+    public static final SparkBaseConfig MOTOR_CONFIG_RIGHT =
+      new SparkMaxConfig()
+        .idleMode(IdleMode.kBrake)
+        .inverted(true)
+        .follow(CAN_ID.LATERATOR_MOTOR_LEFT);
+
+    public static final double MOTOR_P = 0;
+    public static final double MOTOR_I = 0;
+    public static final double MOTOR_D = 0;
+    public static final double MOTOR_F = 0;
+
+    public static final ClosedLoopConfig CLOSED_LOOP_CONFIG_LEFT =
+      MOTOR_CONFIG_LEFT.closedLoop
+        .pidf(MOTOR_P, MOTOR_I, MOTOR_D, MOTOR_F)
+        .outputRange(-1, 1);
+
+    public static final double MAX_MOTION_ALLOWED_ERROR_PERCENT = 0.03;
+
+    public static final double AXIS_MAX_SPEED = 0.1;
+
+    public static final MAXMotionConfig MAX_MOTION_CONFIG_LEFT =
+      CLOSED_LOOP_CONFIG_LEFT.maxMotion
+        .allowedClosedLoopError(MAX_MOTION_ALLOWED_ERROR_PERCENT)
+        .maxAcceleration(0)
+        .maxVelocity(0);
+  }
+
+  public static final class CORAL_RUNNER {
```

Comment:
I think we'll probably want velocity control on this subsystem since we'll want it to be more accurate than the algae intake